Source/SwiftLintFramework/Rules/Metrics/NPathComplexityRule.swift

```diff
@@ -0,0 +1,98 @@
+import SourceKittenFramework
+
+public struct NPathComplexityRule: ASTRule, ConfigurationProviderRule, OptInRule {
+    public var configuration = CyclomaticComplexityConfiguration(warning: 200, error: 1000)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "npath_complexity",
+        name: "NPath Complexity",
+        description: "The number of paths of function bodies should be limited.",
+        kind: .metrics,
+        nonTriggeringExamples: [
+            "func f1() {\nif true {\nfor _ in 1..5 { } }\nif false { }\n}",
```

Comment:
nitpick: please use multiline string literals and properly format the examples. I'd be nice to also avoid `;` and use different values inside a switch.

Reply:
maybe using the same examples from the tests would be nice as they look more like "real code". I'd also keep the comment about the expected complexity inside the example as it's a good way to help people understand what NPath complexity means.